app/views/groups/client_group_list.scala.html

```diff
@@ -70,15 +77,15 @@
             @table(
                 id = Some("client-list-table"),
                 headings = Seq(
-                    Html(checkBox(id = Some("select-all"), value=Some("all")).toString),
+                    Html(checkBox(id = Some("select-all"), value = Some("all")).toString),
```

Comment:
This will clash with my PR, the heading is empty until the Javascript populates the checkbox - although I think we should populate it with some hidden text for non-JS accessibility (and remove it with JS)